compiler_gym/wrappers/validation.py

```diff
@@ -26,9 +29,16 @@ def __init__(
         super().__init__(env)
         self.reward_penalty = reward_penalty
 
-    def step(self, action, observations=None, rewards=None):
-        observation, reward, done, info = self.env.step(
-            action, observations=observations, rewards=rewards
+    def multistep(
+        self,
+        actions: List[ActionType],
+        observation_spaces=None,
+        reward_spaces=None,
```

Comment:
This also needs to accept `observations` and `rewards` arguments for backwards compatibility.